pkgs/swiftgen/swift2objc/lib/src/parser/parsers/declaration_parsers/method_parser.dart

```diff
@@ -0,0 +1,57 @@
+import '../../../ast/_core/shared/parameter.dart';
+import '../../../ast/_core/shared/referred_type.dart';
+import '../../../ast/declarations/compounds/class_declaration.dart';
+import '../../_core/utils.dart';
+import '../declaration_parser.dart';
+
+class MethodParser extends DeclarationParser {
```

Comment:
This class could also be a function (with a bunch of private helper functions in the same file).

Reply:
done